slasher/db/setup_db.go

```diff
@@ -0,0 +1,50 @@
+package db
```

Comment:
if all these are for tests can you suffix these with `_test` ? Otherwise this can get mixed up with the normal slasher code

Reply:
then i have the circular dependency issue

Reply:
`slasher/db/setup_db_test.go` , not sure how this will lead to a circular dependency

Reply:
maybe it wasnt clear, I was referring to suffix the file name, not package

Reply:
but then i cant call those methods from other tests

Reply:
pretty sure you can. We do it all across our repo, calling helpers from other test files

Reply:
in different package? i failed to do it. trying again

Reply:
Or are you referring to other packages ? if thats the case, yeah you couldnt call it. This is a nitpick anyway I dont feel too strongly about it